packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import RatingItem from './RatingItem'
+
+export default class RatingBar extends Component {
```

Comment:
please add some tests for rating bar and item, have a look at the agjs tests for ideas

what do you think about wiring up a snapshot test via jest?  

https://facebook.github.io/jest/docs/snapshot-testing.html